src/types.ts

```diff
@@ -73,8 +78,19 @@ export class PathAliasResolver {
         relativePath = "." + path.sep + relativePath;
       }
 
-      return relativePath.replace(REGEXP_ALL_BACKSLASH, "/");
+      return relativePath.replace(REGEXP_ALL_BACKSLASH, "/"); //;
     } else {
+      if(this.srcPath != this.outPath && requestedModule[0] == "."){
+        let relativeModulePath = fileName.replace(this.srcPath, '');
+
+        const relativeSrcModulePath = path.join(this.srcPath, path.dirname(relativeModulePath), `${requestedModule}.js`);
```

Comment:
Don't work on Windows:
for `requestedModule = "./legacy"`
`relativeSrcModulePath = "Q:\projects\ts-transform-paths\spec\fixtures\Q:\projects\ts-transform-paths\spec\fixtures\actual\dir\legacy.js"`

And legacy test fail:
`
× legacy
      - Expected 'import legacy from "./legacy";
      import * as legacy2 from "./legacy.js";
      import constFile from "./const-file.ts";
      export function legacy() {
          return { legacy, legacy2, constFile };
      }
`

`
      ' to equal 'import legacy from "../../../actual/dir/legacy.js";
      import * as legacy2 from "../../../actual/dir/legacy.js";
      import constFile from "./const-file.ts";
      export function legacy() {
          return { legacy, legacy2, constFile };
      }
      '
`

Reply:
do you have a CI setup in a windows environment?

Reply:
Unfortunately, CircleCI does not support Windows Nano images

Reply:
added a travis yml, passes on windows now